.github/workflows/deliver-ubuntu.yml

```diff
@@ -0,0 +1,110 @@
+name: delivery / ubuntu
+
+on:
+  release:
+    types:
+      - released
+
+jobs:
+    create-ppa:
+      strategy:
+        fail-fast: false
+        matrix:
+          target: [xenial, bionic, focal]
+          include:
+            - target: xenial
+              runner: ubuntu-16.04
+              image: docker://ubuntu:16.04
+            - target: bionic
+              runner: ubuntu-18.04
+              image: docker://ubuntu:18.04
+            - target: focal
+              runner: ubuntu-20.04
+              image: docker://ubuntu:20.04
+      name: create-ppa
+      runs-on:  ${{ matrix.runner }}
+
+      steps:
+          - name: Checkout code
+            uses: actions/checkout@v2
+          - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* || true
```

Comment:
Why is this necessary? Isn't this fetched by actions/checkout? https://github.com/actions/checkout#usage

Reply:
Ya its not, looks like we get the release tag via other methods in our other jobs, copying that format here.